README.md

```diff
@@ -1,31 +1,43 @@
-# nornir_nautobot
 
-[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black)
+<div style="text-align: center;"> 
+  <h1>Nornir Nautobot</h1>
+</div>  
+<p align="center">
+  <a href="https://github.com/nautobot/nornir-nautobot/actions"><img src="https://github.com/nautobot/nornir-nautobot/actions/workflows/ci.yml/badge.svg?branch=main"></a>
+  <a href="https://pypi.org/project/nornir-nautobot/"><img src="https://img.shields.io/pypi/v/nornir-nautobot"></a>
+  <a href="https://pypi.org/project/nornir-nautobot/"><img src="https://img.shields.io/pypi/dm/nornir-nautobot"></a>
+  <br>
+</p>
 
-## Build Status
 
-| Branch  | Status                                                                                                                               |
-| ------- | ------------------------------------------------------------------------------------------------------------------------------------ |
-| develop | [![CI](https://github.com/nautobot/nornir-nautobot/actions/workflows/ci.yml/badge.svg)](https://github.com/nautobot/nornir-nautobot/actions/workflows/ci.yml) |
-## Overview
-
-The nornir_nautobot project intends to solve two primary use cases.
+Nornir-Nautobot is a set of utilities to help interact with Nautobot via Nornir. The nornir_nautobot project intends to solve two primary use cases.
 
 * Providing a Nornir inventory that leverages Nautobot's API.
 * A set of opinionated Nornir plugins.
 
 The set of plugins intend to provide mechanisms to include common networking workflows that will help enable network automation. As an example, there are method to get configurations or test network connectivity. Over time this will include functions to perform actions such as get vlans, neighbors, protocols, etc.
 
-## Getting Started
+# Installation
+
+To install Nornir Nautobot install via Python PIP:
 
 ```shell
 pip install nornir-nautobot
 ```
+## Inventory
+
+The inventory plugin is used to gather inventory from a Nautobot instance. This queries the DCIM endpoint to gather information about the devices.  
+
+[Inventory](inventory/inventory.md)
```

Comment:
I think we can make these links direct to docs, for example `https://docs.nautobot.com/projects/nornir-nautobot/en/latest/inventory/inventory/`. Same goes for the next two.

Otherwise someone clicking them from github will just read the markdown file on github or get a broken link from PyPI I think.

Once the PR is merged I can swap over the domain to docs.nautobot.com.